src/executables/util_cs.c

```diff
@@ -4603,17 +4609,16 @@ memmon (UTIL_FUNCTION_ARG * arg)
 	}
     }
 
-  if (module)
+  if (print_module)
```

Comment:
It's not a flag, so I'm againt applying the same rule with `print_transaction`. I think just `module` is enough. What do you think?